From 2dbbd80a5fb80741729c7cd5027af058b9c08c2c Mon Sep 17 00:00:00 2001
From: Nils Philippsen <nils@redhat.com>
Date: Mon, 8 Jul 2013 17:46:06 +0200
Subject: [PATCH] patch: man-page

Squashed commit of the following:

commit e1915d50b677458127a8ad1c7953ee1d2e2ce250
Author: Nils Philippsen <nils@redhat.com>
Date:   Mon Jul 8 17:44:26 2013 +0200

    xsane.man: update command line options
---
 doc/xsane.man | 29 ++++++++++++++++++++++++-----
 1 file changed, 24 insertions(+), 5 deletions(-)

diff --git a/doc/xsane.man b/doc/xsane.man
index ee363a8..38b453d 100644
--- a/doc/xsane.man
+++ b/doc/xsane.man
@@ -4,6 +4,7 @@
 xsane - scanner frontend for SANE
 .SH SYNOPSIS
 .B xsane
+.RB [ --help | -h ]
 .RB [ --version | -v ]
 .RB [ --license | -l ]
 .RB [ --device-settings
@@ -13,8 +14,9 @@ xsane - scanner frontend for SANE
 .RB [ --viewer | -V ]
 .RB [ --save | -s ]
 .RB [ --copy | -c ]
+.RB [ --multipage | -m ]
 .RB [ --fax | -f ]
-.RB [ --mail | -m ]
+.RB [ --email | -e ]
 .RB [ --no-mode-selection | -n ]
 .RB [ --Fixed | -F ]
 .RB [ --Resizable | -R ]
@@ -25,6 +27,7 @@ xsane - scanner frontend for SANE
 .IR name ]
 .RB [ --display
 .IR d ]
+.RB [ --no-xshm ]
 .RB [ --sync ]
 .RI [ devicename ]
 .SH DESCRIPTION
@@ -121,6 +124,12 @@ and
 .SH OPTIONS
 .PP
 If the
+.B --help
+or
+.B -h
+flag is given xsane displays a short help message and exits.
+.PP
+If the
 .B --version
 or
 .B -v
@@ -128,7 +137,7 @@ flag is given xsane prints a version information, some
 information about gtk+ and gimp version it is compiled
 against and lists the supported file formats, then it exits.
 .PP
-when the
+If the
 .B --license
 or
 .B -l
@@ -161,16 +170,22 @@ or
 flag forces xsane to start in copy mode.
 .PP
 The
+.B --multipage
+or
+.B -m
+flag forces xsane to start in multipage mode.
+.PP
+The
 .B --fax
 or
 .B -f
 flag forces xsane to start in fax mode.
 .PP
 The
-.B --mail
+.B --email
 or
-.B -m
-flag forces xsane to start in mail mode.
+.B -e
+flag forces xsane to start in e-mail mode.
 .PP
 The
 .B --no-mode-selection
@@ -217,6 +232,10 @@ flag selects the X11 display used to present the graphical user-interface
 for details).
 .PP
 The
+.B --no-xshm
+flag forces xsane not to use shared memory images.
+.PP
+The
 .B --sync
 flag requests a synchronous connection with the X11 server.  This is for
 debugging purposes only.
-- 
1.8.3.1

